.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
 
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
-  - conda create --yes -n testenv --file requirements/run.txt --quiet
+  - conda create --yes -n testenv --file requirements/run --quiet
```

Comment:
This needs to be running in the base env since we are testing against things which need access to the conda api.